src/components/WebexInMeeting/WebexInMeeting.scss

```diff
@@ -21,6 +27,7 @@ $shadow-color: rgba(0, 0, 0, 0.2);
   }
 
   .remote-media-in-meeting {
+    height: 100%;
```

Comment:
just flex: 1 is enough if you add display: flex on parent

Reply:
done